dependencies/kyverno/policy/kyverno-rbac.yaml

```diff
@@ -10,9 +10,14 @@ rules:
 - apiGroups:
   - ""
   resources:
+  - pods
+  - secrets
```

Comment:
I had to add secrets to the list of resources due to this error
```
2025-09-04T15:50:51Z ERR github.com/kyverno/kyverno/pkg/background/generate/controller.go:366 > failed to apply generate rule error="rolebindings.rbac.authorization.k8s.io \"konflux-integration-runner-rolebinding\" is forbidden: user \"system:serviceaccount:kyverno:kyverno-background-controller\" (groups=[\"system:serviceaccounts\" \"system:serviceaccounts:kyverno\" \"system:authenticated\"]) is attempting to grant RBAC permissions not currently held:\n{APIGroups:[\"\"], Resources:[\"secrets\"], Verbs:[\"get\" \"list\"]}" logger=background name=ur-7v2tn policy=init-ns-integration trigger=/test
```

Reply:
to make this more robust, should we add a ClusterRoleBinding in `konflux-ci/integration/core` and deploy it together with the ClusterPolicy to bind the `system:serviceaccount:kyverno:kyverno-background-controller` and the ClusterRole `konflux-integration-runner` ? This way if the ClusterRole is updated, we won't face permission problems.